psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -2266,12 +2306,13 @@ PopulationAnalysisCalc::compute_mbis_multipoles(bool print_output) {
 
     // => Post-Processing <= //
  
-    // Atomic density, as defined in Equation 5 of Verstraelen et al. (Negative to account for negative charge of electron)
```

Comment:
Can you provide a small blurb of why we removed this convention?

Reply:
Yes, I removed this to make the code to calculate the radial moments more similar to the original equations (The minus sign may confuse some people). Would you like me to include this comment in the source code?

Reply:
No, I don't think enough people mess with the code that it will interrupt others. I believe it is generally good to explain the change in PRs when moving conventions so that everyone is aware.

Reply:
Great, thanks!